src/scripts/services/dimStoreService.factory.js

```diff
@@ -21,7 +21,8 @@ function StoreService(
   dimManifestService,
   $translate,
   uuid2,
-  dimFeatureFlags
+  dimFeatureFlags,
+  dimDestinyTrackerService
```

Comment:
Without this dangling reference, the tracker service doesn't end up being instantiated.  Google couldn't provide me any guidance on writing what amounts to a background daemon in Angular.